### ISSUE TITLE
Resolve two validating issues

### ISSUE THREAD

**Issue by Brixy** [martignoni/hugo-cloak-email#7]
Just found to validating issues.

1. > The type attribute for the `style` element is not needed and should be omitted.

2. > Element `style` not allowed as child of element p in this context.

I am not quite sure whether No. 2 can be solved/style tag can be moved to a valid position.

**Comment by martignoni** [comment]
Can you please point me to the validator used?

**Comment by Brixy** [comment]
> Can you please point me to the validator used?

I used [validator.w3.org](https://validator.w3.org/).

As far as I know `<style>` is now allowed within body.

Just tested `<body>` > `<style>` which echoes this error message:

> Element style not allowed as child of element body in this context

So the validator is maybe not up to date?!

**Comment by martignoni** [comment]
> So the validator is maybe not up to date?!

Could you try with another (up-to-date) validator, just to be sure?

**Comment by Brixy** [comment]
> Could you try with another (up-to-date) validator, just to be sure?

Ok, now I tested with [this Firefox add-on](https://addons.mozilla.org/en-US/firefox/addon/html-validator/) based on Tidy.

The validating errors mentioned above disappear.

---

By chance I spotted another warning from the plugin. But I guess it is rather a Tidy issue—a least not so important. Anyway:

```html
Text {{< cloakemail address="test@me.com" >}}
```

> Warning: inserting implicit `<p>`

and

> Warning: trimming implicit `<p>`

Warning vanishes when the shortcode is the only ‘child’ of the paragraph:

```html
{{< cloakemail address="test@me.com" >}}
```
